SYCL/InvokeSimd/Spec/simd_size/simd_mismatch_error.cpp

```diff
@@ -2,11 +2,8 @@
 // REQUIRES: gpu && linux
 // UNSUPPORTED: cuda || hip
 //
-// TODO: enable when Jira ticket resolved
-// XFAIL: gpu
-//
 // Check that full compilation works:
-// RUN: %clangxx -fsycl -fno-sycl-device-code-split-esimd -Xclang -fsycl-allow-func-ptr %s -o %t.out | FileCheck %s
+// RUN: !%clangxx -fsycl -fno-sycl-device-code-split-esimd -Xclang -fsycl-allow-func-ptr %s -o %t.out 2>&1 | FileCheck %s
```

Comment:
What does this "!" mean here?

Reply:
Usually if clang errors, the test will fail because it expects exit code 0. Here an error is thrown, so the exit code is non-0, so the test would fail. '!' makes the non-zero exit code a zero, similar to !bool in cpp.

Reply:
Ok, I thought that ! was added to negate the status returned by clang++, but doubted and decided to ask as it is not typical solution.
Typical solution is to use -fsyntax-only and -verify options like here:
https://github.com/intel/llvm/blob/sycl/clang/test/SemaSYCL/esimd-globals-in-sycl-context.cpp#L1
https://github.com/intel/llvm/blob/sycl/clang/test/SemaSYCL/esimd-globals-in-sycl-context.cpp#L18

Reply:
yeah good point, i didnt originally write this test so i didn't consider doing compile only. i responded more in my other comment.